examples/pytorch/rgcn/model.py

```diff
@@ -97,7 +98,11 @@ def __init__(self,
                 self.embeds[str(ntype)] = embed
 
         self.node_embeds = th.nn.Embedding(node_tids.shape[0], self.embed_size, sparse=self.sparse_emb)
-        nn.init.uniform_(self.node_embeds.weight, -1.0, 1.0)
+        if gamma <= 0:
+            nn.init.uniform_(self.node_embeds.weight, -1.0, 1.0)
+        else:
+            emb_init = gamma / embed_size
+            nn.init.uniform_(self.node_embeds.weight, -emb_init, emb_init)
```

Comment:
What is this initialization?

Reply:
I borrow this initialization from KE, this is for DistMult model, RGCN uses the same score function.